sources/packages/web/src/composables/useFormioUtils.ts

```diff
@@ -150,14 +150,16 @@ export function useFormioUtils() {
    * @param forms forms to have the validation executed.
    * @returns false if any form is not valid.
    */
-  const checkFormioValidity = (forms: any[] | FormIOForm[]) => {
-    let isValid = true;
-    forms.forEach((form: any) => {
-      if (!form.checkValidity(undefined, true, undefined, false)) {
-        isValid = false;
+  const checkFormioValidity = async (forms: any[] | FormIOForm[]) => {
+    const promises = forms.concat(forms).map(async (form: any) => {
```

Comment:
why concat here?

Reply:
nice catch! that was a part of my testing. removing